src/entries/popup/pages/walletSwitcher/index.tsx

```diff
@@ -189,6 +196,9 @@ export function WalletSwitcher() {
   const { visibleWallets: accounts, fetchWallets } = useWallets();
   const { data: avatar } = useAvatar({ addressOrName: currentAddress });
   const { trackShortcut } = useKeyboardAnalytics();
+  const { walletsSummary, isLoading: isLoadingSummary } = useWalletsSummary({
```

Comment:
Noticed the ` || []` fallback inside of `useWalletsSummary` is incorrect; looks like a typo